sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -350,14 +364,17 @@ public void setAuthorizationClients(Map<String, AuthorizationClientProperties> a
     }
 
     public boolean isAllowedGroup(String group) {
+        if (this.getUserGroup().getEnableFullList()) {
```

Comment:
I don't think we shoud add the code here

Reply:
If remove this, when enable-full-list is true, all groups will not be listed.